timetracker/entries/urls.py

```diff
@@ -1,12 +1,16 @@
 from django.conf.urls import url
+from django.views.generic import RedirectView
+from django.core.urlresolvers import reverse_lazy
 
 from . import views
+from . views import ClientsView, ProjectsView, EntryView, ClientUpdate, ProjectUpdate
 
 
 urlpatterns = [
-    url(r'^clients/$', views.clients, name='client-list'),
-    url(r'^clients/(?P<pk>\d+)/$', views.client_detail, name='client-detail'),
-    url(r'^entries/$', views.entries, name='entry-list'),
-    url(r'^projects/$', views.projects, name='project-list'),
-    url(r'^projects/(?P<pk>\d+)/$', views.project_detail, name='project-detail'),
+    url(r'^clients/$', ClientsView.as_view(), name='client-list'),
+    url(r'^$', RedirectView.as_view(url=reverse_lazy('client-list')), name='clientroot'),
```

Comment:
pass a permanent=False or permanent=True (depending on the desired behaviour of the redirect) to the as_view() call to remove django's deprecation warning in the console when runserver starts up.